app/javascript/components/upload/DifferentialExpressionStep.jsx

```diff
@@ -49,42 +49,68 @@ export function DifferentialFileUploadForm({
     <div className="row">
       <div className="col-md-12">
         <p className="form-terra">
-          Upload a file with differential expression for a particular clustering and annotation.  (Either format, note auto-detection.  Use tables themed like pre to align columns.  Move tab to top of other files.)
-          <strong>Parsed metadata and clustering files are required before uploading</strong>
+          <p>Upload differential expression (DE) files to enable users to compare genes by DE in cells grouped by type, disease, treatment, and other experimental conditions.  Use long or wide format, one file per annotation.  Comparisons can be one-vs-rest or pairwise.</p>
+          In both formats, headers for "logfoldchanges" and "qval" are required; these let users see log<sub>2</sub>(fold change) and q-value, respectively.  Other metrics like "mean" are optional.
```

Comment:
We want to make sure that the message regarding parsed metadata/clusters is preserved:
```
<strong>Parsed metadata and clustering files are required before uploading</strong>
```

Reply:
Nice catch!  Done in 20b352c.  I've also updated the screenshot, for convenient reference.